GitUI/Editor/FileViewer.cs

```diff
@@ -304,6 +299,10 @@ public int VScrollPosition
 
         // Public methods
 
+        [Description("Use Git coloring with patch commands.")]
+        [DefaultValue(false)]
+        public bool PatchUseGitColoring => showGitWordColoringToolStripMenuItem.Checked || AppSettings.UseGitColoring.Value;
```

Comment:
This is a property, not a method. Please move to the according section above.
The logic needs to be explained (at least to me).

Reply:
[Description("Use Git coloring with patch commands (always for git-word-diff, setting for normal patch).")]